src/test/resources/alma-fix/990050000600206441.json

```diff
@@ -97,35 +97,45 @@
     "source" : {
       "label" : "RVK (Regensburger Verbundklassifikation)",
       "id" : "https://d-nb.info/gnd/4449787-8"
-    }
+    },
+    "id" : "http://rvk.uni-regensburg.de/nt/SK%20110",
+    "label" : "SK 110"
```

Comment:
1. are those the proper URIs (I ask because they don't yield any result)
2. internally there URLs are redirected to their SSL pendant so why not use https directly?